Rufus hamade deployment work

RufusHamade opened the pull request:
Based on RufusHamade-maintenance-fixes-1, 

Contains the docs and scripts required to deply OSMod to a kubernetes cluster.

iislucas commented on README.md:
```diff
@@ -66,12 +93,123 @@ You'll also need to create some (human) users.  The email address of the new use
+  "UNSUBSTANTIAL": {},
+  "LIKELY_TO_REJECT": {},
+  "TOXICITY": {},
+  "TOXICITY_FAST": {}
```
Lets remove toxicity_fast. It's not a good model.

iislucas commented on README.md:
````diff
@@ -66,12 +93,123 @@ You'll also need to create some (human) users.  The email address of the new use
+```
+
+TODO: Need fixes to proxy and osmod:
+ATTACK_ON_PUBLISHER is in proxy's default list and osmod UI, but doesn't work.
````
Yeah, remove it from the list for now.

iislucas commented on README.md:
```diff
@@ -66,12 +93,123 @@ You'll also need to create some (human) users.  The email address of the new use
+
+TODO: Need fixes to proxy and osmod:
+ATTACK_ON_PUBLISHER is in proxy's default list and osmod UI, but doesn't work.
+SUMMARY_SCORE in osmod UI, but doesn't work.
```
summary score should be a field inside each attribute's response, not an attribute in its own right.

iislucas commented on deployments/local/Dockerfile:
```diff
@@ -1,11 +1,14 @@
 COPY . /app/
 
-RUN npm install --unsafe-perm
+RUN npm cache verify && find . -name node_modules -exec rm -Rf {} \; && find . -name package-lock.json -exec rm -Rf {} \;
```
Can you add a comment for why/what you need the find things for. This seems a bit surprising to me.

RufusHamade replied:
Turns out, it wasn't necessary.  Image build issues were actually caused by bad internet connection at office where I did the development.  Running the scripts at home makes it all just work.

I removed the unnecessary code.

iislucas commented:
There's still some pending comments on the PR... are you planning to do more... ? Just wanting to check I'm not accidentally blocking you in a code review?

RufusHamade commented:
Hi Lucas,

I'm just implementing your suggestions.  Actually I have implemented them already - now just testing....

Thanks very much, I really appreciate this feedback, especially as I'm still learning the codebase.

RufusHamade commented on bin/run:
```diff
@@ -0,0 +1,10 @@
+PORT=8080 node ./server.js &
+cd -
+
+sleep 5
```
Sleep was not actually necessary.  I just copied it from the "watch" script, where it *is* necessary.  I removed it now.

RufusHamade commented:
Hi Lucas,

Uploaded a patch with the review markups.  The patch is non-trivial as I've also patched the initial-database.sql script to create the "tags" entries as discussed above, i.e.,
- removed "Attack on publisher"
- set inSummaryScore for all tags.
Also reordered things a bit and removed gaps.  

This means I could remove all the TODOs from the README.

If you have no more comments, then I think we can land this change?

iislucas commented on README.md:
```diff
@@ -66,12 +93,116 @@ You'll also need to create some (human) users.  The email address of the new use
+
+### The Perspective API proxy
+
+OSMod is designed to talk to the Perspective API via a proxy server: the
```
Maybe note also that we plan to remove this and package the proxy with Omod?

RufusHamade replied:
Actually, I decided not to eliminate the proxy entirely, as that is what NYT is currently using, and I didn't want to mess them around too much.  It is now configurable.  I'll queue up that change for review next.

That change rewrites this text fragment a little bit.  So we can update it as you see fit in that review.
